pyad/reverse_mode.py

```diff
@@ -1,4 +1,3 @@
-import math
 import numpy as np
 
 class Reverse:
```

Comment:
Can we name this `Tensor` instead of `Reverse`? So, the way we might use this is like:

**Reverse mode**
```
import pyad.reverse_mode as rev

a = rev.Tensor(123)
y =  rev.sin(a) ** 2
y.backward()  # see my comment below for what this means

print(a.grad)
```

**Forward Mode**
```
import pyad.forward_mode as fwd

a = fwd.Variable('x', 2)
y = fwd.sin(a) ** 2

print(y.d['a'])
```